daisy_workflows/export/disk_export.wf.json

```diff
@@ -21,6 +21,10 @@
       "Value": "200",
       "Description": "size of the export instances disk, this disk is unused for the export but a larger size increase PD read speed"
     },
+    "buffer_disk_type": {
```

Comment:
Use export_instance_disk_type to match the previous variables

Reply:
changed accordingly